pkg/events/events.go

```diff
@@ -0,0 +1,34 @@
+package events
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+)
+
+// Event message format "medik8s <operator shortname> <message>"
+const format = "medik8s %3s %s"
+
+type EventRecorder struct {
+	record.EventRecorder
+	operator string
```

Comment:
I think this needs to be public, otherwise you can´t set it when creating this struct in another package?

Reply:
Nit: IMO might be even nicer to use a method to initialize it instead

Reply:
Agree on both :)